src/librustc/session/config.rs

```diff
@@ -1644,6 +1649,13 @@ pub fn rustc_short_optgroups() -> Vec<RustcOptGroup> {
                                     for the compiler to emit",
             "[bin|lib|rlib|dylib|cdylib|staticlib|proc-macro]",
         ),
+        opt::multi_s(
+            "",
+            "extern-public",
+            "Comma separated list of crates to consider 'public'
```

Comment:
This documentation says comma-separated but the code below seems to not do that, just wanted to double-check!